docker-compose.yml

```
@@ -25,3 +25,18 @@ services:
       - .:/var/www/html
     links:
       - redis
+
+  mysql:
+    image: mysql:latest
+    container_name: mysql
+    volumes:
+      - ./docker/data/db:/var/lib/mysql
+    ports:
+      - ${MYSQL_PORT}:3306
```

```suggestion
      - ${MYSQL_PORT:-3306}:3306
```
p'tit trick pour avoir une valeur par défaut en fallback
https://www.gnu.org/software/bash/manual/html_node/Shell-Parameter-Expansion.html